### PR TITLE
Clean up gpt example

### PR DESCRIPTION
**Pull request by BowenBao**

### CODE REVIEW

**Comment by BowenBao** [comment]
cc @justinchuby @wschin @titaiwangms

File: torch/onnx/_internal/docs/gpt.py [hunk 1]
```diff
@@ -84,7 +85,7 @@ def run_ort(onnx_model, onnx_model_text, pytorch_inputs, pytorch_outputs):
         sess = onnxruntime.InferenceSession(
             onnx_model, providers=["CPUExecutionProvider"]
         )
-        input_names = [v.name for v in onnx_model_text.graph.input]
+        input_names = [i.name for i in sess.get_inputs()]
```
Comment (justinchuby) [inline torch/onnx/_internal/docs/gpt.py:88]: nit: `input`. `i` should only be for iterators

File: torch/onnx/_internal/docs/gpt.py [hunk 2]
```diff
@@ -113,24 +114,28 @@ def test_gpt2_one_shot(model_name):
...
 
     ref_outputs, _ = tree_flatten(model(**inputs, return_dict=False))
-    pth_outputs = run_ort(
+    ort_outputs = run_ort(
         onnx_model, onnx_model_text, (input_ids, attention_mask), ref_outputs
     )
-    for _1, _2 in zip(ref_outputs, pth_outputs):
-        print(_1 - _2)
-        assert torch.allclose(_1, _2)
+    for _1, _2 in zip(ref_outputs, ort_outputs):
```
Comment (justinchuby) [inline torch/onnx/_internal/docs/gpt.py:120]: I would take the opportunity to rename this `_1` `_2` thing. ref_output, ort_output seem reasonable

File: torch/onnx/_internal/docs/gpt.py [hunk 3]
```diff
@@ -113,24 +114,28 @@ def test_gpt2_one_shot(model_name):
...
+        print(_1 - torch.tensor(_2))
+        assert torch.allclose(_1, torch.tensor(_2))
 
 
 def test_gpt2_auto_regressive(model_name):
     # NOTE: auto regressive uses generation algorithms such as greedy search or beam
     # search that involves loops and control flows.
 
+    import logging
```
Comment (justinchuby) [inline torch/onnx/_internal/docs/gpt.py:129]: avoid inline imports?

**Review by justinchuby — APPROVED** [review]
nit

**Review by titaiwangms — APPROVED** [review]
lint

File: torch/onnx/_internal/docs/gpt.py (hunk 2 above)
Comment (BowenBao) [inline torch/onnx/_internal/docs/gpt.py:120]:
```suggestion
    for ref_output, ort_output in zip(ref_outputs, ort_outputs):
```

File: torch/onnx/_internal/docs/gpt.py [hunk 4]
```diff
@@ -113,24 +114,28 @@ def test_gpt2_one_shot(model_name):
...
     ref_outputs, _ = tree_flatten(model(**inputs, return_dict=False))
-    pth_outputs = run_ort(
+    ort_outputs = run_ort(
         onnx_model, onnx_model_text, (input_ids, attention_mask), ref_outputs
     )
-    for _1, _2 in zip(ref_outputs, pth_outputs):
-        print(_1 - _2)
-        assert torch.allclose(_1, _2)
+    for _1, _2 in zip(ref_outputs, ort_outputs):
+        print(_1 - torch.tensor(_2))
```
Comment (BowenBao) [inline torch/onnx/_internal/docs/gpt.py:121]:
```suggestion
        print(ref_output - torch.tensor(ort_output))
```

File: torch/onnx/_internal/docs/gpt.py [hunk 5]
```diff
@@ -113,24 +114,28 @@ def test_gpt2_one_shot(model_name):
...
+        assert torch.allclose(_1, torch.tensor(_2))
```
Comment (BowenBao) [inline torch/onnx/_internal/docs/gpt.py:122]:
```suggestion
        assert torch.allclose(ref_output, torch.tensor(ort_output))
```

File: torch/onnx/_internal/docs/gpt.py (hunk 1 above)
Comment (BowenBao) [inline torch/onnx/_internal/docs/gpt.py:88]:
```suggestion
        input_names = [ort_input.name for ort_input in sess.get_inputs()]
```

**Comment by BowenBao** [comment]
I'm leaving some temporary debug code and other lint issue unchanged .. Need another format solely clean up.

**Pull request merged**